gradle/dependencies.gradle

```diff
@@ -13,15 +13,15 @@ ext {
   // version which we should use in this build
   def mapboxNavigatorVersion = System.getenv("FORCE_MAPBOX_NAVIGATION_NATIVE_VERSION")
   if (mapboxNavigatorVersion == null || mapboxNavigatorVersion == '') {
-      mapboxNavigatorVersion = '123.1.0'
+      mapboxNavigatorVersion = '124.0.0'
   }
   println("Navigation Native version: " + mapboxNavigatorVersion)
 
   version = [
-      mapboxMapSdk              : '10.10.0',
+      mapboxMapSdk              : '10.11.0-beta.1',
       mapboxSdkServices         : '6.10.0-beta.3',
       mapboxNavigator           : "${mapboxNavigatorVersion}",
-      mapboxCommonNative        : '23.2.1',
+      mapboxCommonNative        : '23.3.0-beta.1',
```

Comment:
We also need a compatible Maps SDK version.

Reply:
maps `10.11.0-beta.1` uses common `v23.3.0-beta.1.` (the same as NN v124.0.0)

Reply:
Sorry, I missed this :+1: